internal/cmd/cmds.go

```diff
@@ -43,16 +44,21 @@ func (c *Cmd) Key() string {
 }
 
 func (c *Cmd) Execute(sm *shardmanager.ShardManager) (*CmdRes, error) {
+
+	res := cmdResNil
+	err := errors.ErrUnknownCmd(c.C.Cmd)
 	start := time.Now()
 	if c.Meta == nil {
 		for _, meta := range CommandRegistry.CommandMetas {
 			if meta.Name == c.C.Cmd {
 				c.Meta = meta
+				res, err = c.Meta.Execute(c, sm)
 				break
 			}
 		}
+	} else {
+		res, err = c.Meta.Execute(c, sm)
 	}
```

Comment:
Instead of having this `else`, we can just use the above `if` to populate `Meta` and either throw the error if the command is not found or make sure `Meta` is populated.

This way, we can get rid of `else` and can directly invoke `Execute`. This makes the code more readable.